recipe/patches/0008-Fix-C-17-support-on-windows.patch

```diff
@@ -0,0 +1,32 @@
+From 1e6f1288c5d36c19cf719ca6dd50a25e98f2d6b7 Mon Sep 17 00:00:00 2001
+From: Jiajun Yao <jjyao@anyscale.com>
+Date: Mon, 17 Jan 2022 09:29:52 -0800
+Subject: [PATCH 8/8] Fix C++17 support on windows
+
+Signed-off-by: Jiajun Yao <jeromeyjj@gmail.com>
+---
+ .bazelrc | 3 ++-
+ 1 file changed, 2 insertions(+), 1 deletion(-)
+
+diff --git a/.bazelrc b/.bazelrc
+index 8c15ce5cf..0f621bf3d 100644
+--- a/.bazelrc
++++ b/.bazelrc
+@@ -12,6 +12,7 @@ build:linux --cxxopt="-std=c++17"
+ build:macos --cxxopt="-std=c++17"
+ build:clang-cl --cxxopt="-std=c++17"
+ build:msvc-cl --cxxopt="/std:c++17"
++build:windows --cxxopt="/std:c++17"
```

Comment:
I think this change should be upstreamed to Ray itself

Reply:
Already upstreamed